src/main/java/gregtech/api/capability/impl/MultiblockRecipeLogic.java

```diff
@@ -280,6 +282,12 @@ protected boolean prepareRecipeDistinct(Recipe recipe) {
         return false;
     }
 
+    @Override
+    public boolean prepareRecipe(Recipe recipe) {
+        ((RecipeMapMultiblockController) metaTileEntity).refreshAllBeforeConsumption();
```

Comment:
`prepareRecipeDistinct` probably also needs a refresh call.

Reply:
It already does.